scene/resources/particle_process_material.cpp

```diff
@@ -531,8 +531,8 @@ void ParticleProcessMaterial::_update_shader() {
 
 	code += "\n";
 	code += "void calculate_initial_physical_params(inout PhysicalParameters params, inout uint alt_seed){\n";
-	code += "	params.linear_accel = mix(linear_accel_min, linear_accel_min ,rand_from_seed(alt_seed));\n";
-	code += "	params.radial_accel = mix(radial_accel_min, radial_accel_min,rand_from_seed(alt_seed));\n";
+	code += "	params.linear_accel = mix(linear_accel_min, linear_accel_max, rand_from_seed(alt_seed));\n";
+	code += "	params.radial_accel = mix(radial_accel_min, radial_accel_max, rand_from_seed(alt_seed));\n";
 	code += "	params.tangent_accel = mix(tangent_accel_min, tangent_accel_max,rand_from_seed(alt_seed));\n";
 	code += "	params.damping = mix(damping_min, damping_max,rand_from_seed(alt_seed));\n";
```

Comment:
```suggestion
	code += "	params.tangent_accel = mix(tangent_accel_min, tangent_accel_max, rand_from_seed(alt_seed));\n";
	code += "	params.damping = mix(damping_min, damping_max, rand_from_seed(alt_seed));\n";
```

Reply:
Left them alone to keep the scope small but will add them too

Reply:
Though there's a bunch more style issues like this in the surrounding code, might be out of scope.